bundles/org.openhab.binding.draytonwiser/src/main/java/org/openhab/binding/draytonwiser/internal/handler/HotWaterHandler.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.draytonwiser.internal.handler;
+
+import static org.openhab.binding.draytonwiser.internal.DraytonWiserBindingConstants.*;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.draytonwiser.internal.handler.HotWaterHandler.HotWaterData;
+import org.openhab.binding.draytonwiser.internal.model.DraytonWiserDTO;
+import org.openhab.binding.draytonwiser.internal.model.HotWaterDTO;
+import org.openhab.binding.draytonwiser.internal.model.SystemDTO;
+
+/**
+ * The {@link HotWaterHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Andrew Schofield - Initial contribution
+ * @author Hilbrand Bouwkamp - Simplified handler to handle null data
+ */
+@NonNullByDefault
+public class HotWaterHandler extends DraytonWiserThingHandler<HotWaterData> {
+
+    public HotWaterHandler(final Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected void handleCommand(final String channelId, final Command command) {
```

Comment:
Can you handle a refresh as well?

Reply:
I believe this should be handled by the base class